astro_rcnn.py

```diff
@@ -374,7 +396,7 @@ def detect(directory,mode="detect", outdir = "."):
             # Uncomment below code to visualize as it steps through
             A=1e4
             #im_disp = make_lupton_rgb(image[:,:,0], image[:,:,1], image[:,:,2], stretch=0.000000000001, filename="out.png")
-            #visualize.display_instances((image+A)/100, r[0]['rois'], r[0]['masks'], r[0]['class_ids'], dataset.class_names, r[0]['scores'],save_fig=True)
+            visualize.display_instances((image+A)/100, r[0]['rois'], r[0]['masks'], r[0]['class_ids'], dataset.class_names, r[0]['scores'],save_fig=True)
 
```

Comment:
please change this to the latest Lupton scaling code, remove `A`, and add an option `plot_instances=False` to decide whether to plot `visualize.display_instances` or not.